bot/cogs/git.py

```diff
@@ -131,6 +131,24 @@ async def init(self, ctx):
             )
         )
         await ctx.send(msg)
+    @git.command(help'Prints info on the git pull command')
```

Comment:
```suggestion
    
    @git.command(help='Prints info on the git pull command')
```

Please add a spacing, and a missing equal sign.